schematics/src/ng-add/index.ts

```diff
@@ -6,18 +6,22 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import { ProjectDefinition, WorkspaceDefinition } from '@angular-devkit/core/src/workspace';
+import { WorkspaceDefinition } from '@angular-devkit/core/src/workspace';
+
 import { Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
 import { NodePackageInstallTask } from '@angular-devkit/schematics/tasks';
-
 import { getAppModulePath } from '@schematics/angular/utility/ng-ast-utils';
-import { getWorkspace } from '@schematics/angular/utility/workspace';
-
-import { addModuleImportToRootModule, addPackageToPackageJson, addStyleToTarget } from '../utils';
+import {
+  addModuleImportToRootModule,
+  addPackageToPackageJson,
+  addStyleToTarget,
+  getProjectFromWorkSpace,
+  getWorkspace
+} from '../utils';
 import { hasNgModuleImport } from '../utils/ng-module-imports';
 import { getProjectMainFile } from '../utils/project-main-file';
 import { Schema } from './schema';
-
+import { WorkspaceProject } from '@schematics/angular/utility/workspace-models';
 const NGX_BOOTSTRAP_VERSION = '^6.2.0';
```

Comment:
should read from package.json maybe? create a separate issue, may require `version` script update

Reply:
done issue #6173